gwsurrogate/surrogate.py

```diff
@@ -183,7 +183,7 @@ def __call__(self, q, M=None, dist=None, phi_ref=None,\
 
     if (M is not None) and (dist is not None) and (times is not None) and (units != 'mks'):
     	raise ValueError('passing values of M, dist, and times suggest mks units should be used!')
-
+    
```

Comment:
At this part of the code there's a bunch of checks to make sure the user passes something reasonable. 

I think we should check that if the model is not one of EMRISur1dq1e4 or BHPTNRSur1dq1e4 and `calibration=False` an error is raised. Because this would suggest the user is trying to override the calibration default for a model that doesn't support this parameter in the first place. 

It might also be safer to assign `calibration=None` if the model is not one of EMRISur1dq1e4 or BHPTNRSur1dq1e4. This is just another safeguard that the code doesn't somehow depend on the value of calibration when it shouldn't